api/v1alpha1/servers_types.go

```diff
@@ -104,6 +102,16 @@ type ServersList struct {
 	Items           []Servers `json:"items"`
 }
 
+func (s *ServersList) GetObjectList() []client.Object {
```

Comment:
It feels like these methods can be replaced with a function that takes a `client.ObjectList` as an argument, or a method that takes it as a receiver.

Reply:
I'm not sure how that would work.
```
type ServersList struct {
        metav1.TypeMeta `json:",inline"`
        metav1.ListMeta `json:"metadata,omitempty"`
        Items           []Servers `json:"items"`
}
```
I'm not aware of an existing interface that will let me iterate over Items. Maybe I'll try digging into more of the client code to see if I can find anything.